src/arch/xtensa/boot_loader.c

```diff
@@ -17,6 +17,7 @@
 #include <config.h>
 #include <stddef.h>
 #include <stdint.h>
+#include <cavs/version.h>
```

Comment:
Includes are sorted by increasing visibility. `cavs/*` headers should be first in this file.

Reply:
ack